src/platform/silabs/multi-ota/OTAMultiImageProcessorImpl.cpp

```diff
@@ -422,7 +437,16 @@ void OTAMultiImageProcessorImpl::HandleApply(intptr_t context)
     ChipLogProgress(SoftwareUpdate, "HandleApply: Finished");
 
     // This reboots the device
+#ifdef SLI_SI91X_MCU_INTERFACE
+    // Handle reset logic
+    ChipLogProgress(SoftwareUpdate, "OTA Firmware update completed");
```

Comment:
Let's reduce the logging.

Reply:
This log will be needed once all the TLV processors applied action successfully and before going for reboot.